_includes/shared/markup/ccloud/ccloud-setup-self.adoc

```diff
@@ -1,6 +1,12 @@
-From within the https://www.confluent.io/confluent-cloud/tryfree/[Confluent Cloud UI], click on `Add cloud environment` and name it `kafka-learning`.
-This will keep your learning separate from other Confluent Cloud environments.
+Sign up for https://www.confluent.io/confluent-cloud/tryfree/[Confluent Cloud], a fully-managed Apache Kafka service.
+Then provision Confluent Cloud:
 
-Next click on `Learn` and follow the in-product instructions to launch Kafka and Schema Registry.
-Use the promo code `CC100KTS` to receive an additional $100 free usage on Confluent Cloud (https://www.confluent.io/confluent-cloud-promo-disclaimer[details]).
+1. After you log in to Confluent Cloud, click on `Add cloud environment` and name the environment `learn-kafka`. Using a new environment keeps your learning resources separate from your other Confluent Cloud resources.
 
+2. From the `Billing & payment` section, apply the promo code `CC100KTS` to receive an additional $100 free usage on Confluent Cloud (https://www.confluent.io/confluent-cloud-promo-disclaimer[details]).
+
+3. Click on `LEARN` and follow the instructions to launch a Kafka cluster and to enable Schema Registry.
+
++++++
+<img src="/assets/img/ccloud-home.png" alt="Confluent Cloud" width=800>
```

Comment:
The image renders like this 
![Screen Shot 2021-04-13 at 3 51 52 PM](https://user-images.githubusercontent.com/199238/114612672-6d5cca80-9c70-11eb-8344-15c71563760a.png)

The `img` tag needs to be formatted like this:

`<img src="{{ "/assets/img/ccloud-home.png" | relative_url }}" />`

Looks like this content is reused across other tutorials. So fixing this here and rebasing the other PRs should fix them as well.

Reply:
Thanks for the tip @bbejeck .  Trying this new format in https://github.com/confluentinc/kafka-tutorials/pull/790/commits/0128b0978acbac6c727d41609c58570d56dcba98

Reply:
@bbejeck the image now renders (thanks to your help).